.env

```
@@ -52,6 +71,9 @@ BALANCER_FLASH_FEE_BPS=9
 # Default yield bonus applied to base strategies (basis points).
 DEFAULT_YIELD_BONUS_BPS=18
 # Optional explicit WMATIC/WETH poolId for Balancer yield deposits.
+BALANCER_WMATIC_WETH_POOLID=0x6cfaf40300aa32fa1a3c453f1a6b3ad72037a4af000200000000000000000000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BALANCER_WMATIC_WETH_POOLID key should go before the DEFAULT_YIELD_BONUS_BPS key

(UnorderedKey)